index.html

```diff
@@ -0,0 +1,209 @@
+
+    <header>
+       
+        <figure class="logo">
+            <img src="images/logo.png" alt="">
+        </figure>
+        <div class="header-links">
+            <a href="about.html">About</a>
+            <a href="#program">Program</a>
+            <a href="#join">Join</a>
+            <a href="#sponsor" >Sponsor</a>
+            <a href="#news">News</a>
+            <a href="#initiatives">  </a>
+                <button id="nav-button">Our Initiatives</button> 
+        </div>
+    </header>
+
+    <div class="mobile-menu" id="show-menu">
+         <i class="fa fa-times" id="close-icon" aria-hidden="true"></i>
+       <ul id="hide-list">
+         <li>   <a href="about.html" class="toggle-menu">About</a>      </li> 
+         <li>   <a href="#program" class="toggle-menu">Program</a>  </li> 
+         <li>   <a href="#join" class="toggle-menu">Join</a>        </li> 
+         <li>   <a href="#sponsor" class="toggle-menu">Sponsor</a>  </li> 
+      </ul>
+    </div>
+    
+    
+    <div class="title">
+       
+      <div class="title-container">
+      <i class="fa-solid fa-bars" id="hamburger"></i> 
+
+          <div class="title-details">
+              <h4>"What is a government supposed to do for its people?" </h4>
+                 <div class="gradient">
+             <h1>GOVERNANCE AT THE CENTRE OF AFRICA'S DEVELOPMENT <br> 
+             <br> IBRAHIM GOVERNANCE WEEKEND <br> 2022</h1>
+             </div>
+             </div>
+
+             <div class="mobiletitle">
+             <h1> IBRAHIM GOVERNANCE WEEKEND <br> 2022</h1>
+             </div>
+
+             <div class="event-description">
+             <p>
+                The Mo Ibrahim Foundation is an African foundation, established in 2006 with one focus: the critical importance of governance and leadership for Africa.
+             </p>
+             </div>
+
+             <div class="event-time" id="time">
+             <h2>2022.5.13(FRI) ~ 15(SUN) </h2>
+             <p>Kenyatta International Convention Centre, Nairobi Kenya. </p> 
+        </div>
+      </div>
+    </div>
+</section>
+
+
+<section class="program" id="program" >
+    <div class="program-details">
+
+            <div class="program-header">
+            <h5>Main Program</h5>
+            <div class="underline"></div>
+            </div>
+             
+            <div class="program-cards">
+
+                <div class="card_item">
+                
+                <div class="card_inner">
+                <div class="image"> <img src="images/lecturer.png" alt="lecture"> </div>
+
+                <div class="card_title">Lecture</div>
+            
+
+                <div class="card-text">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+                    tempor.</div>
+    
+                </div>
+                </div>
+
+                <div class="card_item">
+                <div class="card_inner">
+                    <img src="images/exhibition.png" alt="">
```

Comment:
- [ ] Kindly add the alt text in your images in this section, this will help easily figure out what the image was about if it doesn't get displayed.